Modules/CLI/ResampleDTIVolume/itkSeparateComponentsOfADiffusionTensorImage.h

```diff
@@ -61,14 +61,11 @@ class SeparateComponentsOfADiffusionTensorImage
 protected:
   SeparateComponentsOfADiffusionTensorImage();
 
-  void ThreadedGenerateData( const OutputImageRegionType & outputRegionForThread, ThreadIdType threadId ) ITK_OVERRIDE;
+  void DynamicThreadedGenerateData( const OutputImageRegionType & outputRegionForThread) override;
 
-  void GenerateOutputInformation() ITK_OVERRIDE;
-
-  void GenerateInputRequestedRegion() ITK_OVERRIDE;
```

Comment:
Note: I will integrate the commit the replacement of `ITK_OVERRIDE`. That said i will keep the change in a different commit for later integration